lib/main/atom/views/outline/semanticView.ts

```diff
@@ -0,0 +1,64 @@
+import {NavigationTreeComponent} from "./navigationTreeComponent"
+import {NavigationTreeViewModel} from "./semanticViewModel"
+import {initialize} from "./semanticViewPane"
+
+export const SEMANTIC_VIEW_URI = "atomts-semantic-view"
+
+export interface SemanticViewOptions {
+  navTree?: NavigationTreeViewModel | null
```

Comment:
:warning: 'Maybe' qualifier on `navTree` seems unnecessary here since it makes it `... | null | undefined`, which is less than optimal.

Reply:
will be removed